src/components/ebay-carousel/README.md

```diff
@@ -20,15 +20,24 @@ Name | Type | Stateful | Description
 `accessibility-prev` | String | No | aria-label for previous control (default: "Previous Slide")
 `accessibility-next` | String | No | aria-label for next control (default: "Next Slide")
 `index` | String | Yes | 0-based index position
-`items-per-slide` | String | No | automatically fit a number of items for each carousel slide and enable slide controls.
-`gap` | String | No | override the margin between carousel items (default: "16px")
+`items-per-slide` | String | No | automatically fit a number of items for each carousel slide and enable slide controls
+`gap` | String | No | override the margin between carousel items in pixels (default: "16")
 
 ### Additional Attributes for when items-per-slide is set.
 Name | Type | Stateful | Description
 --- | --- | --- | ---
 `accessibility-status` | String | No | status text (default: "Showing Slide {currentSlide} of {totalSlides} - Carousel")
 `accessibility-current` | String | No | pagination current slide text (default: "Current Slide {currentSlide} - Carousel")
 `accessibility-other` | String | No | pagination other slide text (default: "Slide {slide} - Carousel")
+`autoplay` | Boolean | No | adds autoplay controls to the carousel and automatically goes to the next slide every 4 seconds
```

Comment:
Was there any need for config for the `interval` timing?

Reply:
I based the interval on the homepage interval. However it makes it easier to test to make this configurable so I updated the `autoplay` attribute to be either a Boolean or Number. If a number is set that becomes the interval.

Reply:
We should not make the `interval` configurable, as that can lead to abuse. If 4 second is what was standardized then we will stick with it.

Reply:
@senthilp at first I agreed with you on this (which is why my original implementation hardcoded the 4s interval) however after giving it some thought I think this might not be the right approach in this case. The interval for the carousel needs to be finely tuned to the content that is in the carousel such that the user has enough time to properly consume that content. I imagine that could change enough to warrant having it be configurable.

Reply:
@DylanPiercey Agree.